### DISCUSSION TITLE
Stopping a HLS feed

### DISCUSSION THREAD

**Discussion by aortega1** [bluenviron/mediamtx#425] [Questions]
Hello, I'm sorry if this has been covered, I've searched but cannot find a specific topic on this.  

Is there a way to stop a HLS feed?  I'm able to terminate the feed by either closing the page, refreshing the page, or forcing the video tag to point to a new source URL.  The problem is that the HLS feed continues on the RTSP-simple-server log for at least ~1-5 minutes.  

I'm curious if the time is due to the allowed his segments and buffer? (set to 1, and 10 seconds respectively) or if this is another issue I'm seeing?  

In general, is there a URL endpoint I can call to close the connection?  I was thinking I could call something like /camera-1/on /camera-1/off to control the HLS feed.  The 1-5 minute 'overrun' I'm seeing accounts for a few megabytes of data which I'm trying to avoid (cameras are behind a cellular data connection)

I am using HLS, perhaps there is another protocol I should use for this functionality?

Here is a path example

paths:
  camera-1:
    source: rtsp://****:*****@***.***.***.***:554/axis-media/media.amp?camera=1
    sourceProtocol: tcp
    sourceOnDemand: yes

**Comment by aler9** [comment]
Hello, the HLS converter shuts down when nobody requires the stream for 60 seconds. Maybe we can add a parameter to the configuration to make this editable by the user.

**Reply by aortega1** [reply]
Thank you for the response @aler9 , this is good to know .  Yes a parameter would be ideal, currently an open stream will cost me ~180 mb for the duration.

[a comment by an automated account is not shown]
